### DISCUSSION TITLE
Can we get a plugin for Lemmy?

### DISCUSSION THREAD

**Discussion by xylogx** [extesy/hoverzoom#1173] [Ideas]
With the Reddit-pocalypse many people are switching to Lemmy. I noticed that Hover-zoom does not work for Lemmy hosted pics like this one -> https://lemmy.world/pictrs/image/02b119b5-f7a8-4feb-8824-d4d497a36f1c.jpeg

**Comment by extesy** [comment]
Please create a new site request using the "issues" tab. If I understand correctly, Lemmy is an instance of Mastodon which means you are essentially asking to support all Mastodon instances which could be tricky.
